### PR TITLE
lps-99089

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:project-templates

File: modules/sdk/project-templates/project-templates-activator/src/main/resources/archetype-resources/bnd.bnd [hunk 1]
```diff
@@ -1,4 +1,6 @@
 Bundle-Activator: ${package}.${className}
 Bundle-Name: ${artifactId}
 Bundle-SymbolicName: ${package}
-Bundle-Version: ${version}
\ No newline at end of file
+Bundle-Version: ${version}#if (${buildType} != "gradle")
+
+-metayype: *#end
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-activator/src/main/resources/archetype-resources/bnd.bnd:6]: Should be `-metatype: *`

Comment (lawrence-lee) [reply modules/sdk/project-templates/project-templates-activator/src/main/resources/archetype-resources/bnd.bnd:6]: O_o,  darn small font. okay I'll fix it and send it back

**Comment by lawrence-lee** [comment]
ci:stop

**Pull request closed without merging**